logicrunner/goplugin/preprocessor/main.go

```diff
@@ -436,6 +436,8 @@ func generateZeroListOfTypes(parsed *parsedFile, name string, list *ast.FieldLis
 			initializer = "bool(false)"
 		case "string":
 			initializer = `""`
+		case "foundation.Reference":
+			initializer = `*new([64]byte)`
```

Comment:
Why not `initializer = "foundation.Reference([64]byte{})"` ?

Reply:
Just because shorter =)